packages/devtools_shared/test/server/server_api_test.dart

```diff
@@ -36,9 +36,12 @@ void main() {
     );
     expect(response.statusCode, HttpStatus.ok);
     expect(await response.readAsString(), '["debug", "release]');
+    expect(fakeManager.receivedPath, expectedRootPath);
   });
 
-  test('handle deeplink api returns bad request if no root path', () async {
+  test(
+      'handle deeplink api androidBuildVariants returns bad request if no root path',
```

Comment:
nit: in test names, consider using the the enum values directly so the test names don't go stale:
`'${DeeplinkApi.androidBuildVariants} ...'`

here and elsewhere below